### PR TITLE
Re-branded to SAP BTP + a few screenshot updates

### PR DESCRIPTION
**Pull request by raz-korn**

### CODE REVIEW

File: tutorials/appstudio-onboarding/appstudio-onboarding.md [hunk 1]
```diff
@@ -58,11 +62,11 @@ Go to <https://account.hanatrial.ondemand.com> and log in to your SAP Cloud Plat
 
 [ACCORDION-BEGIN [Step: 3](Add SAP Business Application Studio subscription to subaccount)]
 
-In this step, you will add the SAP Business Application Studio subscription to a subaccount. Depending on when you created the SAP Cloud Platform account, this subscription might already be added in the subaccount.
+In this step, you will add the SAP Business Application Studio subscription to a subaccount. Depending on when you created the SAP BTP trial account, this subscription might already be added in the subaccount.
```
Comment (MichaelCzcz) [inline tutorials/appstudio-onboarding/appstudio-onboarding.md:65]: Shouldn't it be "SAP BTP Trial account"?

File: tutorials/appstudio-onboarding/appstudio-onboarding.md [hunk 2]
```diff
@@ -72,7 +76,7 @@ In this step, you will add the SAP Business Application Studio subscription to a
 
     > - Microsoft Azure - Singapore region.
 
-    >When creating an SAP Cloud Platform trial account a **trial** subaccount is generated.
+    >When creating an SAP BTP trial account a **trial** subaccount is generated.
```
Comment (MichaelCzcz) [inline tutorials/appstudio-onboarding/appstudio-onboarding.md:79]: Shouldn't it be "SAP BTP Trial account"?

File: tutorials/appstudio-onboarding/appstudio-onboarding.md [hunk 3]
```diff
@@ -117,13 +121,17 @@ In this step, you will add the SAP Business Application Studio subscription to a
 
     !![legalterms](2020-03 AppStudio Terms__.jpg)
 
-6. A new tab opens and SAP Business Application Studio loads.
+6. A new tab opens and the welcome page for SAP Business Application Studio loads.
+
+    !![sapbusinessapplicationstudiowelcome](BAS_Welcome_.png)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-onboarding/appstudio-onboarding.md:126]: Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.

File: tutorials/appstudio-onboarding/appstudio-onboarding.md [hunk 4]
```diff
@@ -117,13 +121,17 @@ In this step, you will add the SAP Business Application Studio subscription to a
...
 
+7. Click **My Dev Spaces** to open the SAP Business Application Studio dev space manager page.
+    <br>
     >**Bookmark this page!**
 
     >If you create a bookmark to this page, it is easy to get back to SAP Business Application Studio later.
 
-    !![sapbusinessapplicationstudioloaded](2020-03 AppStudio Loaded_.jpg)
+    !![sapbusinessapplicationstudio dev space manager](BAS Dev Space Manager Empty_.png)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-onboarding/appstudio-onboarding.md:134]: Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.

File: tutorials/appstudio-onboarding/appstudio-onboarding.md [hunk 5]
```diff
@@ -58,11 +62,11 @@ Go to <https://account.hanatrial.ondemand.com> and log in to your SAP Cloud Plat
 
 [ACCORDION-BEGIN [Step: 3](Add SAP Business Application Studio subscription to subaccount)]
 
-In this step, you will add the SAP Business Application Studio subscription to a subaccount. Depending on when you created the SAP Cloud Platform account, this subscription might already be added in the subaccount.
+In this step, you will add the SAP Business Application Studio subscription to a subaccount. Depending on when you created the SAP BTP trial account, this subscription might already be added in the subaccount.
 
 1. Click **Enter Your Trial Account** to access the Cloud Foundry environment.
 
-    !![Access SAP Cloud Platform Trial](2020-08 SCP Access Trial_.jpg)
+    !![Access SAP BTP Trial](2020-08 SCP Access Trial_.jpg)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-onboarding/appstudio-onboarding.md:69]: Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.

File: tutorials/appstudio-onboarding/appstudio-onboarding.md [hunk 6]
```diff
@@ -35,21 +35,25 @@ Go to <https://account.hanatrial.ondemand.com> and log in to your SAP Cloud Plat
...
+!![sapbusinessapplicationstudiowelcome](BAS_Welcome_.png)
+
+4. Click **My Dev Spaces** to open the SAP Business Application Studio dev space manager page.
+    <br>
     >**Bookmark this page!**
 
     >If you create a bookmark to this page, it is easy to get back to SAP Business Application Studio later.
 
-    !![sapbusinessapplicationstudioloaded](2020-03 AppStudio Loaded_.jpg)
+    !![sapbusinessapplicationstudio dev space manager](BAS Dev Space Manager Empty_.png)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-onboarding/appstudio-onboarding.md:56]: Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.

File: tutorials/appstudio-onboarding/appstudio-onboarding.md [hunk 7]
```diff
@@ -35,21 +35,25 @@ Go to <https://account.hanatrial.ondemand.com> and log in to your SAP Cloud Plat
...
     >If you receive an error message and your account was not created recently, answer the validation question of the current step, go to the next step in this tutorial, and proceed from there.
 
 2. You might be asked to accept the legal terms. Check the box and click **OK**.
 
     !![legalterms](2020-03 AppStudio Terms__.jpg)
 
-3. A new tab opens and SAP Business Application Studio loads.
+3. A new tab opens and the welcome page for SAP Business Application Studio loads.
 
+!![sapbusinessapplicationstudiowelcome](BAS_Welcome_.png)
```
Comment (MichaelCzcz) [inline tutorials/appstudio-onboarding/appstudio-onboarding.md:48]: Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by raz-korn** [comment]
Fortunately I created a script for this type of update (changing images according to style guide)...

**Pull request merged**